### DISCUSSION TITLE
Blueprint code not executing

### DISCUSSION THREAD

**Discussion by CJWheelz** [microsoft/AirSim#3838] [Support Q&A]
OS: Windows
UE v4.25
AirSim 1.5 (I think)

I am trying to execute very simple blueprint code to control my vehicle's steering and throttle when the game begins. However, it seems like AirSim is overriding these nodes somehow and the car ignores the blueprint commands and just sits there. Is there anyway to use a mixture of Blueprints and programmatic control via the Python APIs? Does this problem have anything to do with changing the parent class to CarPawn when I replaced the default AirSim SUV with my own custom vehicle?

My ultimate goal is to have the car follow a path I draw in Unreal Engine, but then have the throttle controlled via a Python script running simultaneously. Any help is appreciated!

**Comment by zimmy87** [comment]
Where are you trying to execute your blueprint commands from? And which commands are you using to control the vehicle? Would you be able to share the blueprints for your custom vehicle?
